src/commcare_cloud/ansible/roles/postgresql_base/templates/postgresql.conf.j2

```diff
@@ -166,6 +166,10 @@ lc_time = 'en_US.UTF-8'
 
 default_text_search_config = 'pg_catalog.english'
 
+{% if postgresql_idle_in_transaction_session_timeout is defined %}
```

Comment:
I think you can remove this thanks to the change you made in the most recent commit. Again thinking of how to make this look like other settings in this file.
```suggestion
{% if postgresql_idle_in_transaction_session_timeout %}
```

Reply:
Added this change to the PR.